tests/driver_xbee/auto_init_ng_netif/netif_app.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Xbee device initialization
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "kernel.h"
+#include "xbee.h"
+#include "net/ng_nomac.h"
+#include "net/ng_netbase.h"
+
+/* make sure an UART to device is defined in the Makefile */
+#ifndef XBEE_UART
+#error "XBEE_UART not defined"
+#endif
+
+/**
+ * @brief   This is the default baudrate the Xbee modules are programmed to
+ *          when you buy them
+ */
+#define XBEE_BAUDRATE           (9600U)
+
+/**
+ * @brief   MAC layer stack configuration
+ * @{
+ */
+#define STACKSIZE               (KERNEL_CONF_STACKSIZE_DEFAULT)
+#define PRIO                    (0)
```

Comment:
in the test cases the prio does not hurt - as here will never be 'unexpected' other threads...